### ISSUE TITLE
块引搜索结果出现 `<mark>` 标记

### ISSUE THREAD

**Issue by GuangDai** [siyuan-note/siyuan#5650]
### 该问题是否已经被报告过了？ Is there an existing issue for this?

- [X] 我已经搜索了 Issues，没有发现类似问题 
I have searched the existing issues

### 描述问题 Describe the problem

![image](https://user-images.githubusercontent.com/59678726/184779116-5cb59489-7dd2-49f1-8a48-b24e6abc4eff.png)

可以参考下面的视频，换过主题，依旧存在

### 期待的结果 Expected result

和之前的版本一样没有 <mark>

### 截屏或者录屏演示 Screenshot or screen recording presentation

[Uploading 2022-08-16 09-24-03.zip…]()

### 版本环境 Version environment

```markdown
- Version: 2.1.7
- Operating System: ArchLinux
- Browser (if used):
```

### 日志文件 Log file

[siyuan.log](https://github.com/siyuan-note/siyuan/files/9345921/siyuan.log)

**Comment by GuangDai** [comment]
[2022-08-16 09-24-03.zip](https://github.com/siyuan-note/siyuan/files/9346587/2022-08-16.09-24-03.zip)

**Comment by 88250** [comment]
Fix in 103cbf8d15741f319b69ba44676b983613cee4bd
